quickwit-metastore/README.md

```diff
@@ -69,7 +69,7 @@ $ docker-compose up --force-recreate -d
 Once PostgreSQL is up and running, you can run tests including PostgresqlMetastore with the following command.
 
 ```
-$ cargo test --features=postgresql
+$ cargo test --features=postgres
```

Comment:
You'll have to modify this README to refer to the "main" docker-compose: `make docker-compose-up DOCKER_SERVICES=postgres`